### DISCUSSION TITLE
Conditional Formatting-Custom Formula help me, Expert developer

### DISCUSSION THREAD

**Discussion by Aminoragit** [dream-num/univer#2935] [Q&A | 常见问题]
![image](https://github.com/user-attachments/assets/7affc3d3-fa3b-49db-ae34-ae9e1fc106b1)

Coditional Formatting - Custom Formula Not Working like Excel

i Want "if C2 Cell == "Delete" then change background color B2:Z2"
same as "if C3 Cell == "Delete" then change background color B3:Z3"
But formula not working plz help

**Comment by jikkai** [comment]
Duplicate of #2911
